kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/library/maven/search/SearchService.java

```diff
@@ -177,18 +176,6 @@ protected Void call() throws Exception {
         return list;
     }
     
-    // TODO: Return all versions, including snapshots
```

Comment:
This code wasn't used, but it had a TODO, so I don't see the need of removing it, at least as part of this PR.

Reply:
I have restored the method.